redaxo/src/core/lib/util/string.php

```diff
@@ -256,4 +256,17 @@ public static function highlight($string)
         $return = str_replace(["\r", "\n"], ['', ''], highlight_string($string, true));
         return '<pre class="rex-code">' . $return . '</pre>';
     }
+
+    public static function sanitizeHtml(string $html): string
```

Comment:
Da nicht jeder weiß was „sanitize“ bedeutet:

```suggestion
    /**
     * Cleanup the given html string and removes possible malicious codes/markup.
     */
    public static function sanitizeHtml(string $html): string
```